cmd/plugin/cli/version.go

```diff
@@ -0,0 +1,24 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var Version string
+
+func VersionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "version",
+		Args:         cobra.MinimumNArgs(0),
+		Short:        "Show the StorageOS version information",
```

Comment:
does this display StorageOS version or `kubectl-storageos` version?

Reply:
kubectl-storageos version

Reply:
Should probably just read `kubectl storageos version` then IMO

Reply:
I don't understand, sorry